bin/templatemap_generator.php

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
```

Comment:
@Ocramius 

Well, the template map generator will be moved to zend-view, anyway. 

I like your OcraCachedViewResolver but since it requires PHP 7 it is no alternative as long as ZF3 components have a minimum requirement of PHP 5.6.